README.md

```diff
@@ -224,6 +225,10 @@ Type: `Boolean` or `Object`
 
 Default: `undefined` (do not write sourcemaps)
 
+##### other
```

Comment:
I think we should pull the documentation from the through2 repo instead of linking